### PR TITLE
[REVIEW][FEATURE][UWANG-156] - Create Main Layout With Sidebar

### PR DESCRIPTION
**Pull request by maulanafikriash**
#### Description

> Description of features, improvements design.

#### Ticket Reference(s)

[UWANG-156](https://bluehabit.atlassian.net/browse/UWANG-156)

##### UI Changes

> Screenshot or video of UI-related changes
content-mainmin

##### Coverage

> Test coverage screenshot.

N/A

#### Note(s)

> Mention other user or extra note unrelated to the MR here.
@triandamai 

[UWANG-156]: https://bluehabit.atlassian.net/browse/UWANG-156?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by azkifairuz** [comment]
data tablenya udah ada, pull dri feature/UWANG-159

**Comment by maulanafikriash** [comment]
> data tablenya udah ada, pull dri feature/UWANG-159

Oke sip

File: components/MainLayout.vue [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <div :class="sidebarContainerClasses">
+      <!-- <Sidebar/> -->
+    </div>
+
+    <div class="flex-1">
+      <div :class="contentContainerClasses">
+        <!-- <HeaderPage/> -->
+        <main class="overflow-y-auto">
+          <slot>
+            <!-- <StatisticDashboard :statistic="dataStatistic" />
```
Comment (triandamai) [inline components/MainLayout.vue]: klo unused di hapus aja @maulanafikriash

File: components/MainLayout.vue [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+const contentContainerClasses = computed(() => ({
+  'w-full': isSidebarOpen.value, 
+  'ml-0': !isSidebarOpen.value,
+}));
+</script>
+
+<template>
+  <div class="flex h-screen">
+    <div :class="sidebarContainerClasses">
+      <!-- <Sidebar/> -->
```
Comment (triandamai) [inline components/MainLayout.vue]: di bikin `slot` ya

**Review by triandamai — CHANGES_REQUESTED** [review]
cek dulu ya

File: components/MainLayout.vue (hunk 2 above)
Comment (maulanafikriash) [reply components/MainLayout.vue]: Oke udah solved mas

**Review by triandamai — APPROVED** [review]

**Pull request merged**